src/provisioner/provisioner.py

```diff
@@ -183,15 +176,15 @@ def cluster_bootstrap(cortx_conf_url: str = None):
         components = cortx_config_store.get(f'node>{node_id}>components')
         if components is None:
             Log.warn(f"No component specified for {node_name} in CORTX config")
-
+        num_components = len(components)
         mp_interfaces = ['post_install', 'prepare', 'config', 'init']
         for interface in mp_interfaces:
-            for comp_name in components.keys():
+            for j in range(0, num_components):
```

Comment:
Use logical variable name instead of 'j', e.g. comp_number/cnum/ etc

Reply:
Done